README.md

```diff
@@ -50,6 +51,41 @@ MLJ is released under the MIT licensed and sponsored by the [Alan Turing Institu
 
 ---
 
+### Using MLJ
+
+It is a good idea to use a [separate environment](https://julialang.github.io/Pkg.jl/v1/environments/) for MLJ in order to avoid version clashes with other packages you may be using.
+You can do so with
+
+```julia
+julia> using Pkg; Pkg.activate("My_MLJ_env", shared=true)
+```
+
+Installing MLJ is also done with the package manager:
+
+```julia
+julia> Pkg.add(["MLJ", "MLJModels"])
+```
+
+It is important to note that MLJ is essentially a big wrapper providing a unified access to _model providing packages_ and so you will also need to make sure these packages are available in your environment.
+For instance, if you want to use a **Decision Tree Classifier**, you need to have [DecisionTree.jl](https://github.com/bensadeghi/DecisionTree.jl) installed:
+
+```julia
+julia> Pkg.add("DecisionTree");
+julia> using MLJ;
+julia> @load DecisionTreeClassifier
+```
+
+For a list of models and their packages see the [table below](#models-available), we recommend you start with models marked as coming from _mature_ packages such as _DecisionTree_, _ScikitLearn_ or _XGBoost_.
+
```

Comment:
Replace with
"For a list of models and their packages  set the [table below](#models-available) or run `using MLJ; models()`. We recommend ..."